static/templates/subscription.handlebars

```diff
@@ -12,10 +12,14 @@
             <div class="stream-name">{{name}}</div>
             <div class="subscriber-count">
                 {{partial "subscription_count"}}
+                <div class="subscriber-count-hover-content">{{t "Subscriber count" }}</div>
             </div>
+        </div>
+        <div class="bottom-bar">
+            <div class="description" data-no-description="{{t 'No description.'}}">{{{rendered_description}}}</div>
             <div class="stream-message-count">
-                {{#if is_old_stream}}
-                    <i class="icon-vector-comments"></i>
+                {{#if true}}
```

Comment:
This is presumably a typo and should still be `is_old_stream`.

Reply:
yes, It slipped off while testing for this case.